### PR TITLE
Update .gitignore rules for generated example code (for #2479)

### PR DESCRIPTION
**Pull request by TeranIvy**

### CODE REVIEW

File: examples/psyad/lfric/adjoint_partial/.gitignore [hunk 1]
```diff
@@ -0,0 +1,18 @@
+adj_calc_exner_pointwise_mod.F90
```
Comment (arporter) [inline examples/psyad/lfric/adjoint_partial/.gitignore:1]: I think this can just be `*.f90`?

**Review by arporter — CHANGES_REQUESTED** [review]
Thanks for this Iva. I think one of the new files could just have a `*.f90` in it - would you mind checking whether that works OK?
Apart from that, it's all fine.

File: examples/psyad/lfric/adjoint_partial/.gitignore (hunk 1 above)
Comment (TeranIvy) [reply examples/psyad/lfric/adjoint_partial/.gitignore:1]: Unfortunately not, I tried and `git status` sees it. I refreshed the branch with the master and will send the PR back for review.

Comment (arporter) [reply examples/psyad/lfric/adjoint_partial/.gitignore:1]: Oops, I meant `*.F90`! Are you sure it doesn't work?

Comment (TeranIvy) [reply examples/psyad/lfric/adjoint_partial/.gitignore:1]: All of them have `*.F90` extensions. I literally copied and pasted output from `git status` on the master after running examples 😄

Comment (TeranIvy) [reply examples/psyad/lfric/adjoint_partial/.gitignore]: As per PSyclone Telco conversation this morning, I modifed the file to generic `*.F90`. Back to @arporter for another review.

**Review by arporter — APPROVED** [review]
All good now! Thanks Iva. Will proceed to merge.

**Pull request merged**